src/utils/string.ts

```diff
@@ -2,3 +2,8 @@ export function pluralize(num: number, singular: string, plural?: string) {
   const pluralString = plural ?? `${singular}s`;
   return num === 1 ? singular : pluralString;
 }
+
+export function getUtcFromEpoch(epoch: number) {
+  if (epoch == 0) return '--';
```

Comment:
We are using already using the date-fns format function elsewhere in the code. I would say maybe use use that in a utility method and use it everywhere as a standard format to show Date strings.